src/components/SurveyQuestion.js

```diff
@@ -0,0 +1,46 @@
+const SurveyQuestion = ({
+  question,
+  options,
+  selection,
+  isMultiSelect,
+  onOptionClick,
+  onNextQuestion,
+  onPreviousQuestion,
+  onFinish,
+  isLastQuestion,
+  isSingleSelect,
+}) => {
+  return (
+    <div>
+      <h1>{question}</h1>
+      <ul>
+        {options.map((option) => (
+          <li key={option.label}>
+            <button
+              className={
+                { selected: selection.includes(option.label) }
+                  ? "check"
+                  : "selected"
+              }
```

Comment:
This doesn't look right. Did you want to use the `classNames` library here?

Then it should be something like this:

```suggestion
              className={classNames({
                selected: selection.includes(option.label)
              })}
```

Reply:
maybe I forgot to push, it looks like I fixed this part

Reply:
issue already fixed

Reply:
I still see the original code. Please don't resolve the conversation unless you see the fixed code here.